src/styles/ForecastDetails.css

```diff
@@ -0,0 +1,11 @@
+.forecast-details>* {
+  padding-block-start: 10px;
+  font-weight: 600;
+  text-align: center;
+  font-size: 1.5rem;
+}
+
+.forecast-details__temperature>*:first-of-type {
+  padding-block-end: 10px;
```

Comment:
Here I would have put all the forecast details (the ones you wanted to push down) in a separate div with a different class name and then give that one a margin-top to separate it